superset/config.py

```diff
@@ -560,6 +560,9 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 # Cache for datasource metadata and query results
 DATA_CACHE_CONFIG: CacheConfig = {"CACHE_TYPE": "null"}
 
+# Cache for filters state
+FILTER_STATE_CACHE_CONFIG: CacheConfig = {"CACHE_TYPE": "null"}
```

Comment:
Question about this variable's name:

When we start using the KV store for other use cases in the future is the idea that each one will have it's own cache configuration? Or would we have a single cache configuration for all the endpoints implementing a KV store?

If the latter we should rename this config to something more generic, like `KEY_VALUE_STORE_CONFIG`.

Reply:
@betodealmeida each endpoint will have its own cache configuration. We may have different expiration policies depending on the type of information stored.

Reply:
Makes sense!

Reply:
The file system was my original suggestion but during the spike doc review, I was asked to keep null as the default.

Reply:
After more discussions, I changed the code to use the file system as the default backend for Flask Caching.